### PR TITLE
feat(caluma): add new work item order key

### PR DESCRIPTION
**Pull request by Yelinz**

### CODE REVIEW

File: caluma/caluma_workflow/filters.py [hunk 1]
```diff
@@ -201,6 +201,7 @@ class WorkItemOrderSet(BaseFilterSet):
             "deadline",
             "status",
             "slug",
+            "case__document__form__name"
```
Comment (open-dynaMIX) [inline caluma/caluma_workflow/filters.py:204]:
```suggestion
            "case__document__form__name",
```

Or just run black.

**Review by open-dynaMIX — APPROVED** [review]
Otherwise LGTM :+1:

File: caluma/caluma_workflow/filters.py (hunk 1 above)
Comment (Yelinz) [reply caluma/caluma_workflow/filters.py]: used black

**Review by open-dynaMIX — APPROVED** [review]
Awesome :tada:

**Pull request merged**